public/static/docs/user-guide/large-dataset-optimization.md

```diff
@@ -40,9 +40,10 @@ platforms yet. Hard/soft links optimize **speed** and **space** in the file
 system, but may break your workflow since updating hard/sym-linked files tracked
 by DVC in the <abbr>workspace</abbr> causes <abbr>cache</abbr> corruption. These
 2 link types thus require using cache **protected mode** (see the
-`cache.protected` config option in `dvc config cache`). Finally, a 4th "linking"
-alternative is to actually copy files from/to the cache, which is safe but
-inefficient – especially for large files (several GBs or more).
+`cache.protected` config option in `dvc config cache`), which is enabled
```

Comment:
Left this mention to keep the table under it relevant. Could renale it from `no protected mode` to `read-only` though.

Reply:
Hmmm but the sentence above doesn't make much sense now:

> These 2 link types thus require using cache **protected mode**

Reply:
Maybe the whole mention can be removed or changed to just talk about "read-only" and change the term in the table as well.